drivers/scheduler/k8s/specs/kubevirt-multi-vm/dv-cdi-app-ubuntu.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: dv-cdi-app
+spec:
+  pvc:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 17Gi
```

Comment:
does this require 17G of space? Could this be reduced by creating from small base image?

Reply:
Earlier it was 25Gb and compressing it came to 17Gb. Still I can reduce But I need to start building from scratch such as installing applications etc and it will take some time. I'll take up this later.